src/runScript.js

```diff
@@ -1,28 +1,51 @@
 'use strict'
 
-const findBin = require('./findBin')
 const execa = require('execa')
+const pMap = require('p-map')
+
+const findBin = require('./findBin')
 
 module.exports = function runScript(commands, pathsToLint, packageJson, options) {
+    const concurrency = options && options.subTaskConcurrency
```

Comment:
is `subTaskConcurrency` is `0` it will use fallback value `4` I believe. Please use the `readConfigOption` function here as well.